admin_ui/src/components/ChangePassword.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+    <div id="change_password">
+        <div class="inner">
+            <BackButton />
+            <h1>Change password</h1>
+            <form v-on:submit.prevent="changePassword">
+                <label>Old password</label>
+                <input
+                    name="old_username"
+                    type="password"
+                    v-model="oldPassword"
+                />
+
+                <label>New password</label>
+                <input
+                    name="new_password"
+                    type="password"
+                    v-model="newPassword"
+                />
+
+                <label>New password confirmation</label>
+                <input
+                    name="confirm_password"
+                    type="password"
+                    v-model="confirmPassword"
+                />
+
+                <button>Change password</button>
+            </form>
+        </div>
+    </div>
+</template>
+
+
+<script>
+import axios from "axios"
+import BackButton from "../components/BackButton.vue"
+
+export default {
+    data() {
+        return {
+            oldPassword: "",
+            newPassword: "",
+            confirmPassword: ""
+        }
+    },
+    components: {
+        BackButton
+    },
+    methods: {
+        async changePassword() {
+            console.log("Changing password")
+            const payload = {
+                old_password: this.oldPassword,
+                new_password: this.newPassword,
+                confirm_password: this.confirmPassword
```

Comment:
I changed this parameter to `confirm_new_password` in the latest version to be more explicit.